python/paddle/fluid/optimizer.py

```diff
@@ -1180,16 +1180,16 @@ def __init__(self,
                 self._add_average_restore_op(block, param_grad)
 
     def _add_average_apply_op(self, block, param_grad):
-        param = block.clone_variable(param_grad[0])
```

Comment:
This is fixed in develop branch I think,  can you please merge the latest develop branch?